open_schools_platform/organization_management/circles/tests/utils.py

```diff
@@ -60,6 +63,17 @@ def create_data_circle_invite_student(name: str, student_phone: str, parent_phon
     return data
 
 
+def create_data_circle_invite_teacher(name: str, phone: str):
+    data = {
+        "body": {
+            "name": name
+        },
+        "phone": phone,
+        "email": "user@example.com"
```

Comment:
We should think about proper way of testing email services when it will be available.